xbmc/peripherals/bus/android/PeripheralBusAndroid.cpp

```diff
@@ -108,6 +108,12 @@ bool CPeripheralBusAndroid::InitializeProperties(CPeripheral* peripheral)
   return true;
 }
 
+void CPeripheralBusAndroid::Initialise(void)
```

Comment:
it's `Initialise()` in the base class so it needs to be like this in the overriding class too. You can PR a separate change to rename the function if you'd like.